lib/Providers/Qr/BaconQrCodeProvider.php

```diff
@@ -84,7 +78,7 @@ private function getQRCodeByBackend($qrText, $size, ImageBackEndInterface $backe
         $rendererStyleArgs = array($size, $this->borderWidth);
 
         if (is_array($this->foregroundColour) && is_array($this->backgroundColour)) {
-            $rendererStyleArgs = array_merge($rendererStyleArgs, array(
+            $rendererStyleArgs = array(...$rendererStyleArgs, ...array(
```

Comment:
whilst this change is indeed equivalent, I would argue that it is less obvious what is happening now, array_merge is more readable in my opinion. I would say though that replacing `array()` with `[]` would be good to see (i.e. `array_merge($rendererStyleArgs, [`)

Reply:
`array()` vs `[]` is configured in the PHP-CS-Fixer rules. I can change the rules, but it will modify quite some files.